util/src/keys/geth_import.rs

```diff
@@ -80,28 +89,62 @@ pub fn import_geth_key(secret_store: &mut SecretStore, geth_keyfile_path: &Path)
 }
 
 /// Imports all geth keys in the directory
-pub fn import_geth_keys(secret_store: &mut SecretStore, geth_keyfiles_directory: &Path) -> Result<(), ImportError> {
+pub fn import_geth_keys(secret_store: &mut SecretStore, geth_keyfiles_directory: &Path) -> Result<usize, ImportError> {
 	use std::path::PathBuf;
 	let geth_files = try!(enumerate_geth_keys(geth_keyfiles_directory));
+	let mut total = 0;
 	for &(ref address, ref file_path) in &geth_files {
 		let mut path = PathBuf::new();
 		path.push(geth_keyfiles_directory);
 		path.push(file_path);
 		if let Err(e) = import_geth_key(secret_store, Path::new(&path)) {
 			warn!("Skipped geth address {}, error importing: {:?}", address, e)
 		}
+		else { total = total + 1}
 	}
-	Ok(())
+	Ok(total)
 }
 
 
 /// Gets the default geth keystore directory.
-///
-/// Based on https://github.com/ethereum/go-ethereum/blob/e553215/common/path.go#L75
 pub fn keystore_dir(is_testnet: bool) -> PathBuf {
 	path::ethereum::with_default(if is_testnet {"testnet/keystore"} else {"keystore"})
 }
 
+/// Imports all keys from provided file/directory/pattern (tries to interpret `path` in that order)
+pub fn import_keys_pat(secret_store: &mut SecretStore, path: &str) -> Result<usize, ImportError> {
+	// check if it is just one file or directory
+	if let Ok(meta) = fs::metadata(path) {
+		if meta.is_file() {
+			try!(import_geth_key(secret_store, Path::new(path)));
+			return Ok(1);
+		}
+		else if meta.is_dir() {
+			let mut path_buf = PathBuf::new();
+			path_buf.push(path);
+			path_buf.push("*.*");
```

Comment:
supporting paths is fine. windows has powershell if users are desperate to use wildcards.